tools/codegen/models/alert_configuration_api.yaml

```diff
@@ -630,5 +630,632 @@ operations:
+
+            This resource remains under revision and may change.
+        attributes:
+            - string: {}
+              description: Date and time when MongoDB Cloud created the alert configuration. This parameter expresses its value in the ISO 8601 timestamp format in UTC.
+              computed_optional_required: computed
+              tf_schema_name: created
+              tf_model_name: Created
+              api_name: created
+              req_body_usage: omit_always
+              sensitive: false
+              create_only: false
+            - bool:
+                default: false
+              description: Flag that indicates whether someone enabled this alert configuration for the specified project.
+              computed_optional_required: computed
+              tf_schema_name: enabled
+              tf_model_name: Enabled
+              api_name: enabled
+              req_body_usage: omit_always
+              sensitive: false
+              create_only: false
+            - string: {}
+              description: Event type that triggers an alert.
+              computed_optional_required: computed
+              tf_schema_name: event_type_name
+              tf_model_name: EventTypeName
+              api_name: eventTypeName
+              req_body_usage: omit_always
+              sensitive: false
+              create_only: false
+            - string: {}
+              description: |-
+                Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.
+
+                **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
+              computed_optional_required: required
+              tf_schema_name: group_id
+              tf_model_name: GroupId
+              api_name: groupId
+              req_body_usage: omit_always
+              sensitive: false
+              create_only: false
+            - string: {}
+              description: Unique 24-hexadecimal digit string that identifies the alert configuration.
+              computed_optional_required: required
+              tf_schema_name: id
+              tf_model_name: Id
+              api_name: alertConfigId
+              req_body_usage: omit_always
+              sensitive: false
+              create_only: false
+            - list_nested:
+                nested_object:
+                    attributes:
+                        - string: {}
+                          description: Uniform Resource Locator (URL) that points another API resource to which this response has some relationship. This URL often begins with `https://cloud.mongodb.com/api/atlas`.
+                          computed_optional_required: optional
```

Comment:
spotted in other PR (https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3971#discussion_r2605964887), nested attributes are being marked as `Optional`/`Required` in the data source model